team16/'README.md

```diff
@@ -0,0 +1,11 @@
+# Bilder
+
+Här är en bild direkt i projektet:
+
+![character bild](team16/Kevin.png)
```

Comment:
Have you checked the licenses of these pictures? They might require some kind of license note when they are copied/displayed

Reply:
Most files still require their license information to be included, even if it's just a placeholder image. I didn't check if this is the case here, this was just a thought I had when I saw the PR